host/bkps/crons/01_environment_reader_cron.sh

```diff
@@ -0,0 +1,2 @@
+BASH_ENV=/etc/profile.d/automated_bkp_env.sh
```

Comment:
O que faz este script? 01_environment_reader_cron.sh. Isto nao parece cron

Reply:
Este script passa o ficheiro com as variáveis de ambiente para o crontab, uma vez que os crons não conseguiam aceder as variáveis de ambiente de forma interativa. 
A solução foi baseada no workaround disponível neste website https://www.baeldung.com/linux/load-env-variables-in-cron-job .